roles/lib_openshift/library/oc_obj.py

```diff
@@ -1557,7 +1562,8 @@ def run_ansible(params, check_mode=False):
         if state == 'absent':
             # verify its not in our results
             if (params['name'] is not None or params['selector'] is not None) and \
-               (len(api_rval['results']) == 0 or len(api_rval['results'][0].get('items', [])) == 0):
+               (len(api_rval['results']) == 0 or \
```

Comment:
nit: (Not required to change) Parenthesis are easier on the eyes than backslashes.

```python
if ((params['name'] is not None or params['selector'] is not None) and 
        (len(api_rval['results']) == 0 or ...)
        ...):
    ...
```